app/Models/EntryDAOPGSQL.php

```diff
@@ -12,8 +12,13 @@ public function sqlHexEncode($x) {
 
 	protected function autoUpdateDb($errorInfo) {
 		if (isset($errorInfo[0])) {
-			if ($errorInfo[0] === '42P01' && stripos($errorInfo[2], 'entrytmp') !== false) {	//undefined_table
-				return $this->createEntryTempTable();
+			if ($errorInfo[0] === '42P01') {	//undefined_table
```

Comment:
Same here

Reply:
Done in https://github.com/FreshRSS/FreshRSS/pull/2027/commits/036be54ac70b707c1113788a6f9e4146f9981592